pyproject.toml

```
@@ -1,7 +1,71 @@
 [build-system]
-requires = ['setuptools>=40.8.0']
-build-backend = 'setuptools.build_meta'
+requires = ["setuptools>=61.0.0,<69.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Django"
+dynamic = ["version"]
+requires-python = ">= 3.8"
+dependencies = [
+    "asgiref >= 3.6.0, < 4",
+    "backports.zoneinfo; python_version<'3.9'",
```

Nit: If we're following along with master, we generally have normalised with no spaces around version numbers, but spaces in the environment markers: 

```suggestion
    "asgiref>=3.6.0,<4",
    "backports.zoneinfo; python_version < '3.9'",
```

---

Thank you @ngnpope, I copied what we had in `setup.cfg`  so switching to current format is a good idea. Will adjust.